### ISSUE TITLE
Summary not generating!!!

### ISSUE THREAD

**Issue by bjameslondon123** [KaiyangZhou/pytorch-vsumm-reinforce#39]
I have Trained and tested the model but am not able to visualize it. In the summary2video.py there is some typeError popping up stating TypeError: 'KeysViewHDF5' object is not subscriptable.

I have a folder named Videoframes containing several jpg images labelled 000001.jpg and so on....

**Comment by SinDongHwan** [comment]
Hi, @bjameslondon123 

Did you input a h5 path correctly??

**Comment by bjameslondon123** [comment]
Yes i did give the correct results.h5 path, the error is appearing in the summary2video.py file, line 40, in <module> 
key = h5_res.keys()[args.idx]
TypeError: 'KeysViewHDF5' object is not subscriptable.

**Comment by sylvainma** [comment]
@bjameslondon123 did you solve this?

**Comment by pandababyer** [comment]
Hi, I think it is caused by the difference between python2 and python3. Maybe you can rewrite the code 'key = h5_res.keys()[args.idx]'  to 'key = list(h5_res.keys())[args.idx]' in line 40 at summary2video.py

**Comment by hayachiq** [comment]
hi, kindly to help me am facing bellow error
 error: (-215:Assertion failed) !ssize.empty() in function 'cv::resize'

**Comment by SinDongHwan** [comment]
hi, @hayachiq.
Try to check a image variable, after read image.
Maybe you didn't read images.

Good Luck

**Comment by hayachiq** [comment]
thank you for replaying @SinDongHwan 
could you please help me where to find the images ? should they be downloaded with the dataset ?

**Comment by SinDongHwan** [comment]
Hi, @hayachiq.
Yes, You should download a original dataset(images).
And maybe face error about mismatching of image file names. 
so you have to change a code [this line](https://github.com/KaiyangZhou/pytorch-vsumm-reinforce/blob/77f7c93249240d6fa297818c1eefd5c2b51b1747/summary2video.py#L24)
Good Luck~!!

**Comment by sky-16** [comment]
> Hi, @hayachiq.
> Yes, You should download a original dataset(images).
> And maybe face a error about mismatching of image file names.
> So, you have to change a code [this line](https://github.com/KaiyangZhou/pytorch-vsumm-reinforce/blob/77f7c93249240d6fa297818c1eefd5c2b51b1747/summary2video.py#L24)
> Good Luck~!!

I am unable to get from which folder to read images. Where they are placed and what needed to be changed in code.

**Comment by SinDongHwan** [comment]
Hi, @sky-16.
before edit codes, check images file name.

**Comment by sky-16** [comment]
> Hi, @sky-16.
> before edit codes, check images file name.

I have downloaded the dataset mentioned in #40.
Is there any other dataset because in this one I can not find images that need to be read.
Could you tell me in which folder to check for reading images? I followed the default instructions.

**Comment by SinDongHwan** [comment]
Hi, @sky-16.
#40 is not original dataset.
e.g. If you need to TVSum dataset, Go to TVSum dataset home page.
And then Download original dataset.
Others are same.
